tools/vendor_hid_test.py

```diff
@@ -82,42 +86,46 @@ def read_and_print(self) -> int:
     sleep()
     return len(d)
 
-  def get_received_data(self):
+  def get_received_data(self) -> bytes:
     """This combines the data from the received packets, to match the ping
 packets sent."""
     d = b''
-    if len(self.rx_packets) < _PACKETS:
-      raise Exception(f'Insufficent packets received - want {_PACKETS}'
-                      ', got {len(self.rx_packets)}')
-    d += self.rx_packets[-_PACKETS][7:]
-    for p in self.rx_packets[-_PACKETS + 1:]:
+    d += self.rx_packets.pop(0)[7:]
```

Comment:
This line implies that you can only call the function once, and it fails in unexpected ways afterwards. Either we reset here, or keep it open to be called multiple times.

Reply:
Good point.

The code now resets after each call of this function.